predicators/approaches/vlm_planning_prompts/few_shot.txt

```diff
@@ -0,0 +1,38 @@
+You are highly skilled in robotic task planning, breaking down intricate and long-term tasks into distinct primitive actions.
```

Comment:
They (see appendix A3) also seem to provide some more (environment-specific?) information about what to do / not do. E.g.
```
If the object is in sight, you need to directly manipulate it. If the object is not in sight, you need to use primitive skills to find the object
first. If the target object is blocked by other objects, you need to remove all the blocking objects before picking up the target object. At
the same time, you need to ignore distracters that are not related to the task.```

We might want to do this too, if we want to more closely replicate their approach. This additional info seems to guide the VLM to propose the right plan without explicitly telling it which skills to use to do something. 

An analagous thing in BurgerEnv might be something like "You have to use the grill to cook the patty."? It seems hard to figure out what additional information to give -- sometimes it gives away too much? Other times maybe it doesn't.

Reply:
yeah - i feel this is unfair: this is basically trying to move all the motion-level reasoning up to the task-planning level. I think it can kinda work in certain cases, but doesn't feel reliable/trustable...